ext/tidy/tidy.c

```diff
@@ -1058,13 +1058,13 @@ PHP_FUNCTION(tidy_parse_file)
 	obj = Z_TIDY_P(return_value);
 
 	if (!(contents = php_tidy_file_to_mem(ZSTR_VAL(inputfile), use_include_path))) {
-		php_error_docref(NULL, E_WARNING, "Cannot Load '%s' into memory%s", ZSTR_VAL(inputfile), (use_include_path) ? " (Using include path)" : "");
+		php_error_docref(NULL, E_WARNING, "Cannot load \"%s\" into memory%s", ZSTR_VAL(inputfile), (use_include_path) ? " (using include path)" : "");
 		RETURN_FALSE;
 	}
 
 	if (ZEND_SIZE_T_UINT_OVFL(ZSTR_LEN(contents))) {
-		php_error_docref(NULL, E_WARNING, "Input string is too long");
-		RETURN_FALSE;
+		zend_value_error("Input string is too long");
```

Comment:
Can't this use ``zend_arguent_value_error()`` as you are in a PHP_FUNCTION

Reply:
`zend_arguent_value_error()` doesn't really support the wording which is needed here. It should be something like: `Argument ... is a file whose contents is too long`. I can't make it make sense. :)